core/cas-server-core-logging-api/src/main/java/org/apereo/cas/logging/ExceptionOnlyFilter.java

```diff
@@ -0,0 +1,136 @@
+                         final Object p1, final Object p2) {
+        return super.filter(logger, level, marker, msg, p0, p1, p2);
+    }
+
+    @Override
+    public Result filter(final Logger logger, final Level level, final Marker marker, final String msg, final Object p0,
+                         final Object p1, final Object p2, final Object p3) {
+        return super.filter(logger, level, marker, msg, p0, p1, p2, p3);
+    }
+
+    @Override
+    public Result filter(final Logger logger, final Level level, final Marker marker, final String msg, final Object p0,
+                         final Object p1, final Object p2, final Object p3, final Object p4) {
+        return super.filter(logger, level, marker, msg, p0, p1, p2, p3, p4);
+    }
+
+    @Override
+    public Result filter(final Logger logger, final Level level, final Marker marker, final String msg, final Object p0,
+                         final Object p1, final Object p2, final Object p3, final Object p4, final Object p5) {
+        return super.filter(logger, level, marker, msg, p0, p1, p2, p3, p4, p5);
+    }
+
+    @Override
+    public Result filter(final Logger logger, final Level level, final Marker marker, final String msg, final Object p0,
+                         final Object p1, final Object p2, final Object p3, final Object p4, final Object p5,
+                         final Object p6) {
+        return super.filter(logger, level, marker, msg, p0, p1, p2, p3, p4, p5, p6);
+    }
+
+    @Override
+    public Result filter(final Logger logger, final Level level, final Marker marker, final String msg, final Object p0,
+                         final Object p1, final Object p2, final Object p3, final Object p4, final Object p5,
+                         final Object p6, final Object p7) {
+        return filter(logger, level, marker, msg, new Object[] {p0, p1, p2, p3, p4, p5, p6, p7});
+    }
+
+    @Override
+    public Result filter(final Logger logger, final Level level, final Marker marker, final String msg, final Object p0,
+                         final Object p1, final Object p2, final Object p3, final Object p4, final Object p5,
+                         final Object p6, final Object p7, final Object p8) {
+        return super.filter(logger, level, marker, msg, p0, p1, p2, p3, p4, p5, p6, p7, p8);
+    }
+
+    @Override
+    public Result filter(final Logger logger, final Level level, final Marker marker, final String msg, final Object p0,
+                         final Object p1, final Object p2, final Object p3, final Object p4, final Object p5,
+                         final Object p6, final Object p7, final Object p8, final Object p9) {
+        return super.filter(logger, level, marker, msg, p0, p1, p2, p3, p4, p5, p6, p7, p8, p9);
+    }
+
+    @Override
+    public String toString() {
+        return "ExceptionOnlyFilter";
+    }
```

Comment:
Seems like this can be removed, given the default impl in the parent does the same thing, and does avoid a hard-coded name.